manage_proj_delete.php

```diff
@@ -55,11 +55,11 @@
 
 access_ensure_project_level( config_get( 'delete_project_threshold' ), $f_project_id );
 
-$t_project_name = project_get_name( $f_project_id );
-
-helper_ensure_confirmed( lang_get( 'project_delete_msg' ) .
-		'<br />' . lang_get( 'project_name_label' ) . lang_get( 'word_separator' ) . $t_project_name,
-		lang_get( 'project_delete_button' ) );
+$t_message = sprintf( lang_get( 'project_delete_msg' ),
+	project_get_name( $f_project_id ),
```

Comment:
Projekt name has to be sanitized to prevent XSS.

Reply:
fixed